### DISCUSSION TITLE
Cannot install tailwinds on brand new blitz project

### DISCUSSION THREAD

**Discussion by flippyhead** [blitz-js/blitz#2796] [Q&A] [answered]
Created a new blitz project, made no changes, then:

❯ blitz install tailwind

  ERROR  Cannot read properties of undefined (reading 'dots')

 .blitz/recipe-install/node_modules/ink-spinner/build/index.js:12:43

  9:  */
 10: const Spinner = ({ type = 'dots' }) => {
 11:     const [frame, setFrame] = react_1.useState(0);
 12:     const spinner = cli_spinners_1.default[type];
 13:     react_1.useEffect(() => {
 14:         const timer = setInterval(() => {
 15:             setFrame(previousFrame => {

 - Spinner (.blitz/recipe-install/node_modules/ink-spinner/build/index.js:12:43)
 - renderWithHooks (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:6036:18)
 - mountIndeterminateComponent (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:8570:13)
 - beginWork$1 (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:9938:16)
 - Object.invokeGuardedCallbackImpl (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:11563:10)
 - invokeGuardedCallback (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:11740:31)
 - beginWork$$1 (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:15778:7)
 - performUnitOfWork (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:14696:12)
 - workLoopSync (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:14669:22)
 - performSyncWorkOnRoot (.blitz/recipe-install/node_modules/react-reconciler/cjs/react-reconciler.development.js:14265:11)

✕ TypeError: Cannot read properties of undefined (reading 'dots')

**Comment by beerose** [comment, marked as answer]
More information here: https://github.com/blitz-js/blitz/issues/2793
